internal/command/default.provisioners.yaml

```diff
@@ -619,3 +619,56 @@
     - "{{.Uid}}: Or connect your mysql client to \"
     mysql://{{ .State.username }}:{{ .State.password }}@{{ dig .Init.sk "instanceServiceName" "" .Shared }}:{{ .Init.publishPort }}/{{ .State.database }}\""
     {{ end }}
+
+- uri: template://custom-provisioners/kafka-topic
+  type: kafka-topic
+  state: |
+    topic: {{ dig "topic" (print "topic-" (randAlphaNum 6)) .State | quote }}
+  shared: |
+    shared_kafka_instance_name: {{ dig "shared_kafka_instance_name" (print "kafka-" (randAlphaNum 6)) .Shared | quote }}
+  services: |
+    {{ .Shared.shared_kafka_instance_name }}:
+      image: bitnami/kafka:latest
+      restart: always
+      environment:
+        KAFKA_CFG_NODE_ID: "0"
+        KAFKA_CFG_PROCESS_ROLES: controller,broker
+        KAFKA_CFG_LISTENERS: "PLAINTEXT://:9092,CONTROLLER://:9093"
```

Comment:
I see a lot of `9092` "hard-coded, should we have a variable in init section or something?

Reply:
It is the default kafka broker port, but we can make a variable for this, sure

Reply:
Done, there's now a port variable.